### PR TITLE
Add multiple-select v1.2.1 and fix auto-update

### LINKED ISSUE #11297: [Request] Update multiple-select
**Library name:** multiple-select
**Git repository url:** https://github.com/wenzhixin/multiple-select
**npm package name or url (if there is one):** None
**License (List them all if it's multiple):** MIT
**Official homepage:** http://wenzhixin.net.cn/p/multiple-select/docs/
**Wanna say something? Leave message here:** Current cdnjs version: **1.2.0**, Latest available version: **1.2.1** (from Jan 3, 2016)

### PR DESCRIPTION
**Pull request by dakshshah96**
close #11297

Pull request for issue: #11297 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/wenzhixin/multiple-select
 * Official website (optional, not the repository): http://wenzhixin.net.cn/p/multiple-select/docs/
 * NPM package url (optional): none
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 80
   - Count of stars: 1,224
   - Count of forks: 441
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/multiple-select/package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
       {
         "basePath": "",
         "files": [
-          "multiple.select.*"
+          "multiple-select*"
```
Comment (sufuf3) [inline ajax/libs/multiple-select/package.json]: Let's use `"multiple-select*".+(js|css|map)`.
We don't need the file `multiple-select.jquery.json`

Comment (dakshshah96) [inline ajax/libs/multiple-select/package.json]: @sufuf3 Shouldn't it be `"multiple-select*.+(js|css|map)"` instead of `"multiple-select*".+(js|css|map)`?

**Comment by dakshshah96** [comment]
@sufuf3 Done 👍

File: ajax/libs/multiple-select/package.json (hunk 1 above)
Comment (sufuf3) [inline ajax/libs/multiple-select/package.json]: Yes, sorry

**Comment by sufuf3** [comment]
Let's remove the file `multiple-select.jquery.json`, we don't need to add the `.json` file. :)

**Comment by dakshshah96** [comment]
@sufuf3 Oops, sorry. Done 👍

File: ajax/libs/multiple-select/package.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
       {
         "basePath": "",
         "files": [
-          "multiple.select.*"
+          "multiple-select*.+(js|css|map)"
```
Comment (extend1994) [inline ajax/libs/multiple-select/package.json]: We also need .png file. So let us modify this as `multiple-select*.+(js|css|map|png)`.

Comment (dakshshah96) [inline ajax/libs/multiple-select/package.json]: Done 👍

**Comment by dakshshah96** [comment]
@pvnr0082t Don't know why I'm getting a CI error. There's just one directory under `ajax`  — `libs`. Any idea?

**Comment by PeterDaveHello** [comment]
@dakshshah96 i'll take a look

**Comment by PeterDaveHello** [comment]
@PeterDaveHello Looks a little bit wired, I'll update the CI script to print some details so that we'll see if there is something wrong :)

**Comment by dakshshah96** [comment]
@PeterDaveHello Ah, sure 😄

**Comment by PeterDaveHello** [comment]
It looks a little bit weird that the commits here (https://github.com/cdnjs/cdnjs/pull/11327/commits/9c7550200a14fd2caa4629accbde1c0319331301) look different that the CI received(https://github.com/cdnjs/cdnjs/commit/c9c9bbdfde5e3891968baee6dbb8ca20e708c755), rebased and see if there is any different, may be a bug from GitHub.

**Comment by PeterDaveHello** [comment]
Looks good now, I'll dig the webhook and see if it's a bug of GitHub.

**Comment by sufuf3** [comment]
@extend1994 
Please help review this PR, Thank you.

**Comment by dakshshah96** [comment]
@extend1994 @maruilian11 Done 👍

**Comment by maruilian11** [comment]
@dakshshah96 The v1.2.1 files are gone...

**Comment by sufuf3** [comment]
@dakshshah96 Would you want to continue this PR?
Thank you.

**Comment by sufuf3** [comment]
@dakshshah96 
thanks for contribution.
Would you mind working on this and helping improve this PR?
If you have any question, please let us know.

Many thanks.

**Comment by sashberd** [comment]
@sufuf3 @PeterDaveHello @extend1994 Can not rebase this PR to lates master. Could you help me with it or just close this PR

**Comment by extend1994** [comment]
@sashberd Here you go, just take you time.

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Comment by PeterDaveHello** [comment]
(label "PR wait for fixing" should be removed next time :), cc @extend1994 @sashberd )

**Pull request merged**